templates/rhpam75-kieserver-externaldb.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     openshift.io/display-name: Red Hat Process Automation Manager 7.5 managed KIE Server with an external database
     openshift.io/provider-display-name: Red Hat, Inc.
     description: Application template for a managed KIE Server with an external database, for Red Hat Process Automation Manager 7.5 - Deprecated
-    template.openshift.io/long-description: This template defines resources needed to execute Rules and Processes on Red Hat Process Automation Manager KIE server 7.5, including application deployment configuration, secure and insecure http communication, using a external database. Deprecated in 7.5; using the Red Hat Business Automation Operator is recommended.
+    template.openshift.io/long-description: This template defines resources needed to execute Rules and Processes on Red Hat Process Automation Manager KIE server 7.5, including application deployment configuration, secure and insecure http communication, using a external database. Template for Red Hat OpenShift Container Platform version 3.11. Templates are deprecated since Red Hat Process Automation Manager version 7.5; consider using the Red Hat Business Automation Operator.
```

Comment:
I saw you use this a few times: "Template for Red Hat OpenShift Container Platform version 3.11. Templates are deprecated since Red Hat Process Automation Manager version 7.5;"

It sounds a little bit redundant to me. can you please rephrase?

Reply:
@mramendi : I agree with @spolti .